pom.xml

```diff
@@ -505,6 +505,11 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
I haven't checked anyhting this, but I don't know why we need this change. We are already generating javadoc, aren't we? https://javadoc.io/doc/com.hotels.styx/styx-api/0.7.8 , https://javadoc.io/doc/com.hotels.styx/styx-api/0.7.9